sign/sign.go

```diff
@@ -374,14 +374,25 @@ func (s *Signer) VerifyImages(refs ...string) (*sync.Map, error) {
 		}
 
 		if isSigned {
-			unknownRefs = append(unknownRefs, ref)
+			unknownRefsTemp = append(unknownRefsTemp, ref)
 		}
 
 		return true
 	})
 
+	// join unknow refs and remove duplicates
+	unknownRefs = append(unknownRefs, unknownRefsTemp...)
```

Comment:
The appending of `unknownRefsTemp` to `unknownRefs` changes the initial logic in line 363. This will also make the test  fail:

https://github.com/kubernetes-sigs/release-sdk/blob/c76d421c9be0cfcf2abc2395f80b8bd0b4223d8c/sign/sign_test.go#L338-L352